setup.cfg

```diff
@@ -23,12 +23,12 @@ version = 19.5.0
 [options]
 install_requires =
     jinja2
-    requests
+    requests>=2.22.0
```

Comment:
Hopefully this still supports python 2.7

Reply:
Yes it does